### ISSUE TITLE
Training yolov3 with one class

### ISSUE THREAD

**Issue by 20162021** [AlexeyAB/darknet#3959]
So, I need to detect pedestrians in lwir pictures. I have about 50k annotate photos with pedestrian and just with backgrounds. I have found that for only one class you need 4000-5000 iterations but it's just for 2k photos. So, do I need more iterations or 5k would be enough ?

**Comment by Knuust** [comment]
I would observe the loss and mAP rate. If it doesn't decrease/increase anymore, you should stop training.
In general: the bigger the dataset, the more you can train on the set without having problems with overfitting. 
To come to a point: after 5k Iterations you will probably have an ok result. But you can probably improve it by continuing training.
